packages/astro/src/integrations/hooks.ts

```diff
@@ -198,10 +197,6 @@ export async function runHookConfigSetup({
 				addWatchFile: (path) => {
 					updatedSettings.watchFiles.push(path instanceof URL ? fileURLToPath(path) : path);
 				},
-				addDevOverlayPlugin: (entrypoint) => {
-					// TODO add a deprecation warning in Astro 5.
```

Comment:
The TODO here is wrong, this API was unsupported and undocumented before the toolbar actually released, Astro 5 is prime time for fully removing it (the only app that uses it, Sentry, only uses it as a fallback)